<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>activitydetail</class>
 <widget class="QWidget" name="activitydetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>570</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modify_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>570</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>新增</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>570</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>729</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>活动名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="activity_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>729</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>活动景点：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="scenicname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>729</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>活动时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>活动图片：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>活动介绍：</string>
   </property>
  </widget>
  <widget class="QLabel" name="activitypicture">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>631</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="introduction">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>458</y>
     <width>741</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="time">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
